src/main/java/codeu/controller/ProfileServlet.java

```diff
@@ -61,7 +56,8 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
 
     if(requestUrl.length() <= "/profile/".length()){
       // if user navigates to "/profile/" without a specific user
-      request.getRequestDispatcher("/index.jsp").forward(request,response);
+      // TO-DO respond with 404
```

Comment:
nitpick: Use "TODO" instead of "TO-DO". The string "TODO" is actually a programming convention to add comments that help guide the flow of a certain code block. Many editors even have built-in support to highlight comments that start with the string "TODO". Periodically, programmers can scrub the code by searching all files for the string "TODO" and triaging the issues found.

[Google's style guide](https://google.github.io/styleguide/cppguide.html#TODO_Comments), for example, uses this syntax: `// TODO(username): message`.

Reply:
Oh, didn't know about that. Thanks!